cmd/attach/policy/cmd.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package policy
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/aws/tags"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	use     = "policy"
+	short   = "Attach policy to a role"
+	long    = "Attach existing policy to role in the user’s AWS account"
+	example = `  # Attach policy <policy_arn> to role <role_name>
+  rosa attach policy --policy-arn=<policy_arn> --role-arn=<role_name>`
+)
+
+var args struct {
```

Comment:
@marcolan018 Could you take a look at this example of how we're trying to remove the need for global `args` struct. It would be good to try and follow the same pattern here: https://github.com/openshift/rosa/pull/1777#discussion_r1512694837